tests/Unit/TransferInformation/CustomerCreditTransferInformationTest.php

```diff
@@ -1,17 +1,18 @@
 <?php
 
-namespace tests;
+namespace Digitick\Sepa\Tests\Unit\TransferInformation;
 
 use Digitick\Sepa\TransferInformation\CustomerCreditTransferInformation;
+use PHPUnit\Framework\TestCase;
 
-class CustomerCreditTransferInformationTest extends \PHPUnit_Framework_TestCase
+class CustomerCreditTransferInformationTest extends TestCase
 {
     /**
      * Tests whether the EndToEndId equals the name if no other identifier was supplied
      */
     public function testEndToEndIndentifierEqualsName()
     {
-        $information = new CustomerCreditTransferInformation(100, 'DE12500105170648489890', 'Their Corp');
+        $information = new CustomerCreditTransferInformation('100', 'DE12500105170648489890', 'Their Corp');
```

Comment:
the change of the amount to string is not ok as this is not the interface used up to now, the test should also work woth an int or a float in this place

Reply:
According to the phpDoc, the amount should be a string.
https://github.com/php-sepa-xml/php-sepa-xml/blob/8f9814bc7c97eda669e93848df3dcdb28ca7debb/lib/Digitick/Sepa/TransferInformation/CustomerCreditTransferInformation.php#L28

When you use static analysis (like I do on my project) it require to pass a string. So this should be the tested value.

I can update the phpDoc to `string|float|int` and test the three value if you want. I think it's the best to do.

Reply:
Hehe, good catch.

I think we should update the docs. I'd like to use typehints but this won't work here.

Can you tell more about you static analysis I would be interested in what tool you are running.

Regards

Reply:
I updated the doc and added tests then.

I'm using https://github.com/phpstan/phpstan and I love it.
I heard about psalm too but never tried.